modules/hosted-control-planes-concepts-personas.adoc

```diff
@@ -21,6 +21,8 @@ hosted control plane:: An {product-title} control plane that runs on the managem
 
 hosting cluster:: See _management cluster_.
 
+managed cluster:: A cluster that the hub cluster manages in Red Hat Advanced Cluster Management. A managed cluster is not the same thing as a _management cluster_.
```

Comment:
We should mention MCE as well here. Maybe point to the definition of managed cluster elsewhere as well ,and what it means to "manage" a cluster from an ACM/MCE perspective?

Reply:
@zanetworker I updated the entry to add more context and add a link to the page in the ACM docs that defines "managed cluster". Let me know what you think when you can.